src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/InputAbbreviationAsWordInNameTypeSnakeStyle.java

```diff
@@ -0,0 +1,57 @@
+/*
+AbbreviationAsWordInName
+allowedAbbreviations = ORDER, OBSERVATION, UNDERSCORE, TEST
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abbreviationaswordinname;
+
+public class InputAbbreviationAsWordInNameTypeSnakeStyle {
+    // violation below '.* name 'FLAG_IS_FIRST_RUN' must contain no more than '4' .* cap.*.'
+    public boolean FLAG_IS_FIRST_RUN = false;
+
+    // violation below '.* name 'HBCKL_LOCK_PATH' must contain no more than '4' .* cap.*.'
+    private int HBCKL_LOCK_PATH;
+
+    Boolean[] BOOL_VALS = { false, true };
+
+     // violation below '.* name '__DEMOS__TESTS_VAR' must contain no more than '4' .* cap.*.'
```

Comment:
I pretty sure you can remove leading `.*`. Please do

Reply:
Done